tests/test_patches.py

```diff
@@ -27,9 +27,9 @@
 
     # resources/action.py
     ServiceAction.__init__: {'b8b759abbe8fbfa9bad332b8ce8d30f55daf97f3'},
-    ServiceAction.__call__: {'3e84efcb2badef725b998e1af27a4a6f6f1aa2ed'},
-    BatchAction.__call__: {'8839f17467d16f98e4587fa750d74c9007dea134'},
-    WaiterAction.__call__: {'57d4f3f74e7e774e3aeb13630292fa9183a790f8'},
+    ServiceAction.__call__: {'f3cb58a5e36bf3355723c69ec244990180a2d5bc'},
```

Comment:
This seems caused only by changes in formatting, https://github.com/boto/boto3/commit/e2492c98c9c0f7787aa193a6709ff98ace09a5d8.